src/Middleware/OutputCaching/src/OutputCachePolicyBuilder.cs

```diff
@@ -275,6 +275,14 @@ public OutputCachePolicyBuilder Cache()
         return this;
     }
 
+    /// <summary>
+    /// Adds a policy to ignore the HOST header in the cache key.
+    /// </summary>
+    public OutputCachePolicyBuilder NoVaryByHost()
```

Comment:
```suggestion
    public OutputCachePolicyBuilder DisableVaryByHost()
```

Reply:
Adding this suggestion to the api-review issue. I went with "No" because of "NoCache" in the same class.